microsoft-edge/extensions-chromium/developer-guide/sidebar.md

```diff
@@ -38,7 +38,9 @@ Extensions can optionally use the sidebar API to show a custom UI in the Microso
 
 Features of the Sidebar API include:
 
-* The sidebar remains open while navigating between tabs.
+* The sidebar remains open while navigating between tabs.<!-- todo: fix/reconcile -->
+
+  Note: Currently, the sidebar is not automatically displayed again when the user switches to a tab in which the sidebar was previously open.<!-- todo: if behavior changes, update or remove statement -->
```

Comment:
Instead of "Note:", what about using "Known issue:"?

```suggestion
  Known issue: the sidebar is not automatically displayed again when the user switches to a tab in which the sidebar was previously open, see the [tracking bug on the microsoft/MicrosoftEdge-Extensions repo](https://github.com/microsoft/MicrosoftEdge-Extensions/issues/142).<!-- todo: if behavior changes, update or remove statement -->
```

Reply:
thx, spectrum of options, from minor to heavy:
* "Note:" text
* `> [!NOTE]` alert ([depr.](https://review.learn.microsoft.com/en-us/help/platform/markdown-reference?branch=main#alerts-note-tip-important-caution-warning)) 
* "Known issue:" text
* "Known issue" minimal heading
* "Known issue: Foo is bar" detailed heading

Reply:
result 2x: 
```
Known issue: The sidebar is not automatically displayed again when the user 
switches to a tab in which the sidebar was previously open; see 
[Issue 142](https://github.com/microsoft/MicrosoftEdge-Extensions/issues/142).
<!-- todo: update -->
```